src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js

```diff
@@ -926,20 +926,20 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
                   marginBottom: 10,
                   minWidth: 300,
                 }}
-              >
+                >
                 <Row style={{ width: '100%' }}>
                   <Row style={{
                     alignItems: 'center',
                     justifyContent: 'space-between',
                     marginLeft: 10,
                     width: BOX_PLOT_RATIO,
                   }}
-                  >
+                       >
                     <span style={{
                       color: theme.greyScale3,
                       fontSize: '1.2rem',
                     }}
-                    >
+                          >
```

Comment:
For the record, some of those closing tags realigning to the first prop would look less weird if the prop goes onto a new line. ☝️

Reply:
My auto formatting is consistently inconsistent, I'm going to look into it next week

Reply:
hm it's something at eslint I'm fixing on my commit. Don't sweat it ;) I gotchu!